src/chordsymbol.ts

```diff
@@ -715,7 +733,7 @@ export class ChordSymbol extends Modifier {
     } /* CENTER_STEM */ else {
       x = note.getStemX() - this.getWidth() / 2;
     }
-    L('Rendering ChordSymbol: ', this.text, x, y);
+    L('Rendering ChordSymbol: ', this.textFont, x, y);
```

Comment:
is this intentional?

Reply:
Yes, `this.text` was only used in the log function so I thought it must be a bug. `this.textFont` was used overall.